packages/taquito/src/operations/operation-emitter.ts

```diff
@@ -206,6 +206,10 @@ export abstract class OperationEmitter {
               ...op,
               period: currentVotingPeriod?.voting_period.index,
             };
+          case OpKind.FAILING_NOOP:
```

Comment:
FYI operation-emitter is no longer used and was removed in my most recent PR merge.